virttest/qemu_vm.py

```diff
@@ -4315,6 +4315,21 @@ def check_block_locked(self, value):
                     return block['locked']
         return False
 
+    def get_block_virtual_size(self, dev_dict={}):
+        """
+        Get block virtual-size of specified block device
+        from monitor's info block command.
+
+        :param dev_dict: define device dict from block info.
+        :return: Matched block virual size.
+        """
+        blocks_info = self.monitor.info("block")
+        for dict in blocks_info:
+            if dev_dict.values()[0] in dict.values():
```

Comment:
This will not work in Python3, If you want get a value from dict, please use it's key
`Python 3.6.2 (default, Sep  1 2017, 12:03:48) 
[GCC 7.1.1 20170802 (Red Hat 7.1.1-7)] on linux
Type "help", "copyright", "credits" or "license" for more information.`
`>>> d1 = {"a":1, "b":2, "c":3}`
`>>> d1.values()`
`dict_values([1, 2, 3])`
`>>> d1.values()[0]`
`Traceback (most recent call last):`
`  File "<stdin>", line 1, in <module>`
`TypeError: 'dict_values' object does not support indexing`

Reply:
ok, thanks, will update it~